### DISCUSSION TITLE
Custom aspect ratio

### DISCUSSION THREAD

**Discussion by ArkadiyEpanov** [ultralytics/yolov5#2916] [Q&A]
Hello, I have a question related to the aspect ratio. When I start object detection with the 1280 setting, the frame height is automatically determined as 960. In the logs, I get the resolution in the reverse order of 960x1280. The videos from the camera come in 1280 by 1024. Am I doing something wrong? I can't find how to change the ratio settings for my cam. I would be grateful for any advice!

**Comment by glenn-jocher** [comment]
@Selar007 in YOLOv5 you specify the size of the largest image dimension only. The smaller dimension is handled automatically to meet padding constraints and retain a resize-aspect ratio of 1.0.
```
python train.py --img 1280
```

**Reply by ArkadiyEpanov** [reply]
thank you, I set python train.py --img 1280, how does the number 960 appear? In my case cam resolution is 1280x1024. 

Is it possible to somehow change the output from opencv to a resolution other than 640x480?

**Reply by glenn-jocher** [reply]
The smaller dimension is handled automatically to meet padding constraints and retain a resize-aspect ratio of 1.0.

**Reply by glenn-jocher** [reply]
You can specify any --img-size you want, i.e.
```
python train.py --img 256
python train.py --img 320
python train.py --img 640
python train.py --img 1280
python train.py --img 1920
```

**Reply by linmeon** [reply]
for example,all image size is 1280*720,can i use --img 1280,720 to train yolo?

**Reply by glenn-jocher** [reply]
```
python train.py --img 1280
```
